core/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -533,26 +533,29 @@ public void deleteLocks(ZooReaderWriter zk, String path, String txidStr)
     }
   }
 
+  // Follow on ticket to rework exception handling to
+  // coincide with changes to FateCommand. At the moment, user has no feedback of why something
+  // failed without having to go check server logs.
   @SuppressFBWarnings(value = "DM_EXIT",
       justification = "TODO - should probably avoid System.exit here; "
           + "this code is used by the fate admin shell command")
   public boolean checkGlobalLock(ZooReaderWriter zk, ServiceLockPath path) {
     try {
       if (ServiceLock.getLockData(zk.getZooKeeper(), path) != null) {
-        System.err.println("ERROR: Manager lock is held, not running");
+        log.error("ERROR: Manager lock is held, not running");
```

Comment:
If changing these to log messages, including "ERROR" in the message is redundant.